automl/beta/src/test/java/com/example/automl/BatchPredictTest.java

```diff
@@ -35,7 +35,7 @@
 public class BatchPredictTest {
   private static final String PROJECT_ID = System.getenv("AUTOML_PROJECT_ID");
   private static final String BUCKET_ID = PROJECT_ID + "-lcm";
-  private static final String MODEL_ID = "VCN0000000000000000000";
+  private static final String MODEL_ID = "TEN0000000000000000000";
```

Comment:
Really - that's it?

Reply:
Yea, the video version of AutoML only supports CSV so I guess they recently added a proper check for the file type, but since we use fake model_id's for testing, just switched to a different automl product that does support the file type.